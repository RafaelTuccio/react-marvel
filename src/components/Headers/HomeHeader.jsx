import logo from '../assets/logo.svg'

const HomeHeader = () => {
    return(
        <header>
            <img src={logo} alt="logo marvel" />
            <h1>EXPLORE O UNIVERSO</h1>
            <span>Mergulhe no dominío deslumbrante de todos os personagens clássicos que você ama - e aqueles que você descobrirá em breve!</span>
        </header>
    )
}

export default HomeHeader;