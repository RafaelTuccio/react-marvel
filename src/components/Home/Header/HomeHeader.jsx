import logo from '../../assets/logo.svg'


const HomeHeader = () => {
    return(
        <header className="header-home">
            <div className="logo-maior">
                <img src={logo} alt="logo marvel" />
                <h1>EXPORE O UNIVERSO</h1>
                <span className="header-subtitulo">Mergulhe no domínio deslumbrante de todos os personagens clássicos que você ama - e aqueles que você descobrirá em breve!</span>
            </div>
        </header>
    )
}

export default HomeHeader;