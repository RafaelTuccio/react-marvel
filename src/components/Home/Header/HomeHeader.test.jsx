import { render, screen } from '@testing-library/react';
import HomeHeader from './HomeHeader';


describe("<HomeHeader />", () => {
    it("deve renderizar normalmente", () => {
        const { debug } = render(<HomeHeader />)
        debug();
        expect(screen.getByRole("img", {name: "logo marvel"}));
        
        
    })

    it("deve dar match com o snapshot", () => {
        const { container } = render(<HomeHeader />);
        expect(container.firstChild).toMatchSnapshot();
    })
})