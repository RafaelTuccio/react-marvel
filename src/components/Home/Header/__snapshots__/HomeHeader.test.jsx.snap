// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomeHeader /> deve dar match com o snapshot 1`] = `
<header
  class="header-home"
>
  <div
    class="logo-maior"
  >
    <img
      alt="logo marvel"
      src="logo.svg"
    />
    <h1>
      EXPLORE O UNIVERSO
    </h1>
    <span
      class="header-subtitulo"
    >
      Mergulhe no domínio deslumbrante de todos os personagens clássicos que você ama - e aqueles que você descobrirá em breve!
    </span>
  </div>
</header>
`;
